src/librustc_mir/transform/const_prop.rs

```diff
@@ -988,8 +1023,103 @@ impl<'mir, 'tcx> MutVisitor<'tcx> for ConstPropagator<'mir, 'tcx> {
             | TerminatorKind::GeneratorDrop
             | TerminatorKind::FalseEdges { .. }
             | TerminatorKind::FalseUnwind { .. } => {}
-            //FIXME(wesleywiser) Call does have Operands that could be const-propagated
-            TerminatorKind::Call { .. } => {}
+            TerminatorKind::Call { ref mut args, .. } => {
+                // FIXME(felix91gr) Finish this
```

Comment:
I think we can remove this now 🙂

Reply:
Yas! I'm uploading a renewed version asap, with more doc comments as well